PersonDataRequest/PersonDataRequest/Controller/DataViewController.swift

```diff
@@ -26,26 +26,50 @@ class DataViewController: UIViewController {
     
     func queryUserData() {
         guard let selectedName = selectedName else { return }
-        let genderUrl = "https://api.genderize.io/?name=\(selectedName)"
+        let genderUrl = "https://api.genderizeeee.io/?name=\(selectedName)" // make "eee" mistake in URL
```

Comment:
Please remove this "eee" since it was only for testing purposes

Reply:
Removed